docs/site/src/demos/lists/SingleLineList.js

```diff
@@ -0,0 +1,226 @@
+// @flow weak
+
+import React, { Component } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import {
+  List,
+  ListItem,
+  ListItemAvatar,
+  ListItemIcon,
+  ListItemSecondaryAction,
+  ListItemText,
+} from 'material-ui/List';
+import Avatar from 'material-ui/Avatar';
+import FolderIcon from 'material-ui/svg-icons/folder';
+import IconButton from 'material-ui/IconButton';
+import { LabelCheckbox } from 'material-ui/Checkbox';
+import Layout from 'material-ui/Layout';
+import SvgIcon from 'material-ui/SvgIcon';
+import Text from 'material-ui/Text';
+
+const SqureIcon = (props) => (
+  <SvgIcon {...props} viewBox="3 3 18 18">
+    <path d="M3,3V21H21V3" />
+  </SvgIcon>
+);
+
+const styleSheet = createStyleSheet('SingleLineList', (theme) => ({
+  root: {
+    flexGrow: 1,
+    maxWidth: 752,
+  },
+  container: {
+    width: '100%',
+  },
+  demoContainer: {
+    margin: '0 auto',
+    maxWidth: 360,
+  },
+  demo: {
+    background: theme.palette.background.paper,
+    border: `solid ${theme.palette.text.divider}`,
+    borderWidth: '1px 1px 0',
+    marginBottom: 16,
+    maxHeight: 160,
+    overflow: 'hidden',
+    position: 'relative',
+    width: '100%',
+    '&:before': {
+      backgroundImage: `linear-gradient(45deg, ${theme.palette.background.contentFrame} 25%, transparent 25%),
+                        linear-gradient(-45deg, ${theme.palette.background.contentFrame} 25%, transparent 25%)`,
+      backgroundSize: '20px 20px',
+      backgroundPosition: '10px -10px',
+      bottom: 0,
+      content: '""',
+      display: 'block',
+      height: 10,
+      position: 'absolute',
+      width: '100%',
+      zIndex: 99,
```

Comment:
@oliviertassinari You mean `z-index`? Basically to be over all elements in demo but not overlap toolbar when scrolling.